pkg/relayer/proxy/proxy.go

```diff
@@ -41,10 +47,14 @@ type relayerProxy struct {
 	// which is needed to check if the relay proxy should be serving an incoming relay request.
 	sessionQuerier sessiontypes.QueryClient
 
-	// providedServices is a map of the services provided by the relayer proxy. Each provided service
+	// advertisedRelayServers is a map of the services provided by the relayer proxy. Each provided service
 	// has the necessary information to start the server that listens for incoming relay requests and
-	// the client that proxies the request to the supported native service.
-	providedServices map[string][]*ProvidedService
+	// the client that relays the request to the supported native service.
+	advertisedRelayServers RelayServersMap
+
+	// nativeServiceListenAddresses is a map of the native services server's listen addresses
```

Comment:
Ditto one of my previous comments. Are these the proxy addresses?

Reply:
When mentioning `native services` it's referring to the "Anvil" like server address, I apologize if the naming is totally misleading :/